src/decisionengine/framework/engine/tests/fixtures.py

```diff
@@ -167,9 +167,9 @@ def de_server_factory(request):
         server_proc.start()
 
         # Ensure the channels have started
-        logger.debug(f"DE Fixture: Waiting on startup state is_set={server_proc.de_server.startup_complete.is_set()}")
-        server_proc.de_server.startup_complete.wait(timeout=3)
-        logger.debug(f"DE Fixture: startup state is_set={server_proc.de_server.startup_complete.is_set()}")
+        logger.debug(f"DE Fixture: Timed wait on startup state: is_set={server_proc.de_server.startup_complete.is_set()}")
+        server_proc.de_server.startup_complete.wait(timeout=5)
```

Comment:
Alas, I saw some hanging in the past.... but that was a ways back.  We may have solved some of the startup sync issues since then.  I've no objection to dropping the timeout here.

The one a bit further down (the `block-while` bits) absolutely causes a number of hangs with some of the tests.  But that has more to do with testing the code paths where the modules may not be fully setup/implemented/etc.

Reply:
Let's try it without the wait on line 171 and see what happens.

Reply:
Ahem, I mean let's try the wait *without the timeout*.

Reply:
Done